pkg/tracking/store/sql/tags.go

```diff
@@ -80,3 +80,136 @@ func (s TrackingSQLStore) setTagsWithTransaction(
 
 	return nil
 }
+
+func (s TrackingSQLStore) SetTag(
```

Comment:
Wait, I'm confused now, did you try and port

```py
    def set_tag(self, run_id, tag):
        """
        Set a tag on a run.

        Args:
            run_id: String ID of the run.
            tag: RunTag instance to log.
        """
        with self.ManagedSessionMaker() as session:
            tag = _validate_tag(tag.key, tag.value)
            run = self._get_run(run_uuid=run_id, session=session)
            self._check_run_is_active(run)
            if tag.key == MLFLOW_RUN_NAME:
                run_status = RunStatus.from_string(run.status)
                self.update_run_info(run_id, run_status, run.end_time, tag.value)
            else:
                # NB: Updating the run_info will set the tag. No need to do it twice.
                session.merge(SqlTag(run_uuid=run_id, key=tag.key, value=tag.value))
```

`FAILED .mlflow.repo/tests/store/tracking/test_sqlalchemy_store.py::test_update_run_name` is failing because `tag.key == MLFLOW_RUN_NAME` is not ported.

Reply:
Oh, I thought that part had not been implemented in other endpoints, now I realized that the `MLFLOW_RUN_NAME` value was aliased to `TagRunName` in the Go code!

I also just realized that the `test_set_tag` and `test_update_run_name` were failing all along. I'll take a shot at implementing the missing logic now.